### DISCUSSION TITLE
I need to show the Info window after click on the marker

### DISCUSSION THREAD

**Discussion by subhashkLa** [alexandra-c/leaflet-tracking-marker#12] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/31767342/164184787-a824068d-33c9-4251-9761-3c05145aba62.png)

Something like this

Is there any way??

This library is really great and helpful to me. If I get this Info tab part It will be great

**Comment by alexandra-c** [comment, marked as answer]
Hello @subhashkLa ,

You can find an example in **react-leaflet** documentation, here https://react-leaflet.js.org/docs/example-popup-marker/
You just have to use the **Popup** component from **react-leaflet**  inside your marker, something like this:

```
    <LeafletTrackingMarker
      icon={icon}
      position={[lat, lng]}
      previousPosition={prevPos}
      duration={1000}
    >
      <Popup autoPan={false}>{"something"}</Popup>
    </LeafletTrackingMarker>
```

**Comment by MinSomai** [comment]
<img width="439" alt="Screenshot 2024-04-03 at 5 35 46 PM" src="https://github.com/alexandra-c/leaflet-tracking-marker/assets/21338691/efd8a9bf-5a9b-44a1-8cc6-18ce645c0716">

```js
      <LeafletTrackingMarker
        icon={icon}
        position={[lat, lng]}
        previousPosition={prevPos}
        duration={1000}
        keepAtCenter={true}
      >
        <Popup>Hello</Popup>
      </LeafletTrackingMarker>
    </>

```

Getting this error when I put Popup. @alexandra-c

**Reply by MinSomai** [reply]
my bad

you have to import Popup from `react-leaflet` and not `leflet`
